dora/core/server/master/src/main/java/alluxio/master/job/CopyJobFactory.java

```diff
@@ -57,6 +57,7 @@ public Job<?> create() {
     boolean partialListing = options.hasPartialListing() && options.getPartialListing();
```

Comment:
in line 52, since we are passing the src string to ufsiterator can we do a clean up the path by give it a new AlluxioURI(src)?

Reply:
I would prefer to do it on the client side to ensure it's clean otherwise throw an exception